substrate-airgapped/Cargo.toml

```diff
@@ -9,15 +9,17 @@ edition = "2018"
 codec = { package = "parity-scale-codec", version = "1.3.5", default-features = false }
 
 # Substrate (hopefully these can be removed in the future)
-sp-runtime = { version = "2.0.0", default-features = false }
-sp-core = { version = "2.0.0", default-features = false }
+sp-runtime = { version = "2.0.1", default-features = false }
+sp-core = { version = "2.0.1", default-features = false }
+primitive-types = { version = "0.7", default-features = false, features = ["rustc-hex", "serde"] }
+anyhow = "1"
```

Comment:
For now can we keep this as a dev dep? I see it has no [deps itself](https://github.com/dtolnay/anyhow/blob/master/Cargo.toml), which is good for keeping our dep tree slim; And it is no_std compat., but I am not totally convinced we need it.

Reply:
It's a good top-level error handling lib that actively tries to do as little as possible; it plays nice with `thiserror` too. People often use `anyhow` in binaries and `thiserror` in library code.

You sure you prefer to do without it? :)

Reply:
removed.